src/benchmarks/micro/corefx/System.Text.Json/Utf8JsonReader/Perf.Comments.cs

```diff
@@ -41,18 +39,14 @@ public partial class Utf8JsonReaderCommentsTests
         public void Setup()
         {
             _jsonPayload = Encoding.UTF8.GetBytes(s_testCases[TestCase]);
-
-            if (MultiSegment)
-            {
-                _jsonPayloadSequence = GetSequence(_jsonPayload, SegmentSize);
-            }
+            _jsonPayloadSequence = GetSequence(_jsonPayload, 100);
         }
 
         [Benchmark]
-        public void Utf8JsonReaderCommentParsing()
+        public void EmptyLoop()
         {
             var state = new JsonReaderState(options: new JsonReaderOptions { CommentHandling = CommentHandling });
-            Utf8JsonReader reader = MultiSegment ?
+            Utf8JsonReader reader = !SingleSegment ?
```

Comment:
You an as well swap expressions and remove `!`

Reply:
Will do.